src/styles/blocks/product__card.scss

```diff
@@ -0,0 +1,124 @@
+.product__card {
+  box-sizing: border-box;
+  border-radius: 5px;
+  border: 1px solid rgb(243, 243, 243);
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  transition: all 0.3s ease-in-out;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &:hover .product__name {
+    color: #34568b;
+  }
+
+  .product__image {
+    margin: 32px 20px 40px 20px;
+    height: $product-image-height;
+    background: $mac-img center no-repeat;
+    background-size: contain;
+  }
+
+  .product__info {
+    display: flex;
+    flex-direction: column;
+    padding: 0 16px 16px 16px;
+
+    .product__name {
+      padding-bottom: 4px;
+      font-size: 12px;
+      font-weight: 500;
+      line-height: 18px;
+    }
+
+    .product__code {
+      padding-bottom: 16px;
+      color: $product-att-color;
+      font-size: 10px;
+      font-weight: 400;
+      line-height: 14px;
+    }
+
+    .product__review-box {
+      width: 100%;
+      display: flex;
+      justify-content: space-between;
+      padding-bottom: 24px;
+
+      .product__stars {
+        display: flex;
+
+        .product__star {
+          height: 16px;
+          width: 16px;
+          background-image: url(../images/star.svg);
+          background-position: center;
+          background-repeat: no-repeat;
+          background-size: contain;
+
+          &:not(:last-child) {
+            margin-right: 4px;
+          }
+
+          &--highlighted {
+            background-image: url(../images/star-active.svg);
+          }
+        }
+      }
+
+      .product__review-counter {
+        font-size: 10px;
+        font-weight: 400;
+        line-height: 14px;
+      }
+    }
+
+    .product__price-box {
+      width: 100%;
+      display: flex;
+      justify-content: space-between;
+      padding-bottom: 16px;
+
+      .product__price-label {
+        color: $product-att-color;
+        font-size: 12px;
+        font-weight: 400;
+        line-height: 18px;
+      }
+
+      .product__price-value {
+        color: rgb(6, 11, 53);
+        font-size: 16px;
+        font-weight: 700;
+        line-height: 18px;
+      }
+    }
+
+    .buy-button {
+      border-radius: 5px;
+      border: 1px solid $buy-button-color;
+      box-sizing: border-box;
+      text-align: center;
+      text-decoration: none;
+      text-transform: uppercase;
+
+      color: rgb(255, 255, 255);
+      font-size: 14px;
+      font-weight: 700;
+      line-height: 16px;
+
+      background-color: $buy-button-color;
+      padding: 12px 0;
+      width: 100%;
+      transition: all 0.15s ease-in-out;
+
+      &:hover {
+        background-color: rgb(255, 255, 255);
+        color: $buy-button-color;
+      }
```

Comment:
It's better to transition only the properties that actually change. In this case, you're changing 'background-color' and 'color' on hover, so you should specify these properties in the 'transition' property instead of using 'all'.